clang/test/CodeGen/ubsan-bitfield-conversion.c

```diff
@@ -0,0 +1,61 @@
+// RUN: %clang -fsanitize=implicit-bitfield-conversion -target x86_64-linux -S -emit-llvm -o - %s | FileCheck %s --check-prefixes=CHECK,CHECK-BITFIELD-CONVERSION
```

Comment:
I'd like to see some testcases that run in C++ mode.

Reply:
I will add some testcases later today!

Reply:
Added!